src/any_agent/frameworks/agno.py

```diff
@@ -40,18 +34,19 @@ def _get_model(self, agent_config: AgentConfig):
         )
 
     async def _load_agent(self) -> None:
+        if not agno_available:
+            msg = "You need to `pip install 'any-agent[agno]'` to use this agent"
+            raise ImportError(msg)
+        if self.managed_agents:
+            msg = "Managed agents are not yet supported in Agno agent."
+            raise NotImplementedError(msg)
+
         if not self.managed_agents and not self.config.tools:
             self.config.tools = [
                 search_web,
                 visit_webpage,
             ]
-        tools, mcp_servers = await wrap_tools(
-            self.config.tools, agent_framework=AgentFramework.AGNO
-        )
-        # Add to agent so that it doesn't get garbage collected
-        self._mcp_servers = mcp_servers
-        mcp_tools = [mcp_server.tools for mcp_server in mcp_servers]
-        tools.extend(mcp_tools)
+        tools, _ = await self._load_tools(self.config.tools)
```

Comment:
Is it necessary to pass self.config.tools as a parameter, or can _load_tools access it via self.config.tools since it's already a member function?

Reply:
I leave tools as argument because it is also used in the managed_agents like `self._load_tools(managed_agent.tools)`

Reply:
Then maybe you can leave it as optional with default value `None` and use `self.config.tools` in case it is `None`

Reply:
um. no strong opinion but I think it is more clear to leave it as a required arg